src/debugger/debug_workflow.ts

```diff
@@ -202,6 +214,32 @@ export class WorkflowContextDebug extends DBOSContextImpl implements WorkflowCon
     return this.startChildWorkflow(wf, ...args);
   }
 
+  invokeOnConfig<T extends object>(targetCfg: ConfiguredClass<T>): InvokeFuncsConf<T> {
```

Comment:
We only have one invoke now
```suggestion
  invoke<T extends object>(targetCfg: ConfiguredClass<T>): InvokeFuncsConf<T> {
```

Reply:
Yes but not this way, it needed merged onto the other function.
I did grep for invokeOnConfig, so not sure how 3 occurrences got missed, or how it compiles for that matter, but it did and it did.